### PR TITLE
Feature/localized attributes

### PR DESCRIPTION
**Pull request by petarcurkovic**
Hi @d4be4st !

I've added support for I18n translations.

### CODE REVIEW

**Comment by d4be4st** [comment]
I cant tell how this will behave with booleans...
Please test that boolean is returned unchanged

Also you dint have to upgrade the version as we didnt relese the gem yet.

**Comment by petarcurkovic** [comment]
Thanks for info!
It's fixed now.

File: lib/enumerations/value.rb [hunk 1]
```diff
@@ -58,13 +58,21 @@ def define_attributes_getters
         next if respond_to?(key)
 
         self.class.send :define_method, key do |locale: I18n.locale|
-          I18n.t(key, scope: [:enumerations, self.class.name.underscore, symbol],
-                      default: @attributes[key],
-                      locale: locale) unless @attributes[key].nil?
+          if [String, Symbol].include?(@attributes[key].class)
```
Comment (d4be4st) [inline lib/enumerations/value.rb]: imas `.is_a?(String)`

Comment (d4be4st) [inline lib/enumerations/value.rb]: or you can write something like:

```
case @attribute[key]
when String, Symbol then translate_attribute(key, locale)
else @attributes[key]
end
```

File: test/translation_test.rb [hunk 2]
```diff
@@ -17,12 +17,42 @@ def test_translated_value_with_i18n_locale
     assert_equal 'Nacrt', translated_name
   end
 
+  def test_translated_value_with_i18n_locale_and_custom_locale
+    status = Status.find(:draft)
+
+    I18n.locale = :hr
```
Comment (d4be4st) [inline test/translation_test.rb]: I was thinking something about this:

``` ruby
I18n.locale = :hr
assert_equal 'Nacrt',  status.name
I18n.locale = :en
assert_equal 'Draft', status.name
```

**Pull request merged**